src/libs/Accessibility/index.js

```diff
@@ -0,0 +1,48 @@
+import {useEffect, useState, useCallback} from 'react';
+import {AccessibilityInfo} from 'react-native';
+import _ from 'lodash';
+import moveAccessibilityFocus from './moveAccessibilityFocus';
+
+const useScreenReaderStatus = () => {
+    const [isScreenReaderEnabled, setIsScreenReaderEnabled] = useState(false);
+    useEffect(() => {
```

Comment:
This looks like it can be simplified:

```js
useEffect(() => AccessibilityInfo.addEventListener('screenReaderChanged', setIsScreenReaderEnabled), []);
```